### PR TITLE
Add MSTeams icons

### LINKED ISSUE #6: Decide on or create necessary icons
Right now we will be needing several different icons:
- Marketplace icon
- Bot profile picture
- Channel header icon
- Meeting started icon
Screenshot from 2020-07-17 11-42-18

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Update icons to use MSTeams meetings icons.

#### Screenshots
Screenshot from 2020-10-07 13-16-12
Screenshot from 2020-10-07 13-16-15
Screenshot from 2020-10-07 13-16-33
Screenshot from 2020-10-07 13-16-36
Screenshot from 2020-10-07 13-48-06
Screenshot from 2020-10-07 13-48-29

#### Ticket Link
Fix #6

### CODE REVIEW

**Comment by larkox** [comment]
@aaronrothschild Pleaste take a look at this to approve the icons. We don't know if there is any legal limitation for us to use the official icon or about creating or own icon.

File: webapp/src/components/post_type_mstmeetings/post_type_mstmeetings.tsx [hunk 1]
```diff
@@ -108,10 +107,9 @@ export default class PostTypeMSTMeetings extends React.PureComponent<Props> {
                             target='_blank'
                             href={props.meeting_link}
                         >
-                            <i
-                                style={style.buttonIcon}
-                                dangerouslySetInnerHTML={{__html: Svgs.VIDEO_CAMERA_3}}
-                            />
+                            <i style={style.buttonIcon}>
+                                <Icon/>
```
Comment (mickmister) [inline webapp/src/components/post_type_mstmeetings/post_type_mstmeetings.tsx:111]: Was the `Icon` component used elsewhere before being introduced here?

**Review by mickmister — APPROVED** [review]

**Review by catalintomai — APPROVED** [review]

File: webapp/src/components/post_type_mstmeetings/post_type_mstmeetings.tsx (hunk 1 above)
Comment (larkox) [reply webapp/src/components/post_type_mstmeetings/post_type_mstmeetings.tsx:111]: Yes, it was used when registering the channel header icon.

**Review by aaronrothschild — APPROVED** [review]
I checked on MS Site, they have no guidelines but they have icons that look just like this one.  LGTM.

**Comment by larkox** [comment]
Merging without QA since we are testing on release test.

**Pull request merged**